src/utils/daos_hdlr.c

```diff
@@ -2834,18 +2774,28 @@ fs_copy_hdlr(struct cmd_args_s *ap)
 	struct fs_copy_args	fa = {0};
 	int			src_str_len = 0;
 	int			dst_str_len = 0;
-
+	char			*name = NULL;
+	char			*dname = NULL;
+ 	mode_t			tmp_mode_dir = NULL; 
```

Comment:
(style)  trailing whitespace
(style)  code indent should use tabs where possible
(style)  please, no space before tabs